docs/internal-developers/block-client-apis/checkout/checkout-api.md

```diff
@@ -1,9 +1,12 @@
 # Checkout API interface <!-- omit in toc -->
 
+<<<<<<< HEAD
 **Note on migration:** We are in the process of moving much of the data from contexts into data stores, so this portion of the docs may change often as we do this. We will endavour to keep it up to date while the work is carried out
```

Comment:
It looks like a merge went wrong here. `<<<<<<< HEAD` shouldn't be there and the paragraph `**Note on migration:** We are in the process of ...` currently appears twice in this doc.